From 34b938345d77fe2dada2c4ed8c28559520704316 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 8 Oct 2014 14:15:39 -0400
Subject: [redhat] kabi: Adding symbol __node_distance (arch/x86/mm/numa.c)

Message-id: <1412777739-19786-1-git-send-email-poros@redhat.com>
Patchwork-id: 97221
O-Subject: [RHEL7.1 PATCH] kabi: Adding symbol __node_distance (arch/x86/mm/numa.c)
Bugzilla: 1062669
RH-Acked-by: Don Zickus <dzickus@redhat.com>

This patch is kabi fix. Only adding symbol for ppc64, because in original request time
    symbol wasn't exported.
    Inconsistency is caused by this patch: http://patchwork.lab.bos.redhat.com/patch/79399/

    Original request info
    BZ: 1062669

    Description:
    ============
    These are standard for working with NUMA hardware like UV.

Signed-off-by: Petr Oros <poros@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/__node_distance b/redhat/kabi/kabi-module/kabi_ppc64/__node_distance
new file mode 100644
index 0000000..dc1b9dd
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/__node_distance
@@ -0,0 +1,2 @@
+#0-
+0x46c47fb6 __node_distance vmlinux EXPORT_SYMBOL
-- 
1.7.1